### ISSUE TITLE
CheStarter returns a 400 error code to fabric8-wit when the error is a 401 http error

### ISSUE THREAD

**Issue by hectorj2f** [redhat-developer/che-starter#204]
CheStarter is returning a 400 Bad Request error code but the underlying exception is 401 Unauthorized due to OSO token expired.

You can search in dsaas-preview logs for status:500 or for instance req_id: 8136f380-7196-4d21-b050-a0dd948c4800 in Kibana

related to: https://github.com/fabric8-services/fabric8-wit/issues/1538

**Comment by ibuziuk** [comment]
@hectorj2f yeah, this is because of the fact that Bad Request response is used when `HttpClientErrorException` occurs. Need to provide better handling

[1] https://github.com/redhat-developer/che-starter/blob/master/src/main/java/io/fabric8/che/starter/exception/GlobalExceptionHandler.java#L54

**Comment by hectorj2f** [comment]
@ibuziuk what is the state of this issue ?

**Comment by ibuziuk** [comment]
@hectorj2f haven't started working on it and did not add to the current sprint. Is it critical ?

**Comment by hectorj2f** [comment]
@ibuziuk I just want it to be sure it wasn't solved 👍

**Comment by ibuziuk** [comment]
@hectorj2f nope, it was not - I will try to find some time to address it though ;-)
